src/other/pages/what-is-new/story.js

```diff
@@ -86,6 +86,6 @@ export const WhatIsNew = () => {
     ${contact}
   `;
 
-  render(template, wrapper);
-  return wrapper;
-};
+    render(template, wrapper);
+    return wrapper;
+  });
```

Comment:
Why do we have all these whitespace changes? Dont we use the same prettier config? Or is that eslint? Please check your IDE

Reply:
I created the file. After only one person edited the file. I have nothing changed in eslint or prettier since then.
If I was the only person to edit this files, my guess would be that there wouldn't be whitespace changes at all. So I think the person who edited the files must has different settings.

Reply:
The whitespace is justified. Look on how the structure of the code changed.